packages/firestore/test/unit/local/indexeddb_persistence.test.ts

```diff
@@ -1108,6 +1136,12 @@ describe('IndexedDb: canActAsPrimary', () => {
       expect(await getCurrentLeaseOwner()).to.not.be.null;
     });
   });
+
+  it('obtains lease if forceOwningTab is set', () => {
+    return withPersistence('clientA', async (persistence, platform, queue) => {
+      await withForcedPersistence('clientB', db2 => Promise.resolve());
```

Comment:
Can we verify somehow that `clientA` is not useable anymore? We could invoke `runTransaction` or something.